CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # ChangeLog
 
+## 3.8.2
```

Comment:
In the past we have left the version out until right before we release. That way we can decide the release version later on.